wordcab_transcribe/main.py

```diff
@@ -50,6 +51,15 @@
 async def startup_event():
     """Startup event handler."""
     logger.debug("Starting up...")
+
+    user_platform = retrieve_user_platform()
+    if user_platform != "Ubuntu" or user_platform != "Linux":
+        logger.warning(
+            "You are not running the application on Ubuntu or Linux.\n"
```

Comment:
Did you mean MacOS or Linux

Reply:
No, I think it should work with other Linux distros too.

Or should we restrict to Ubuntu?

Reply:
The warning should indicate that we tested this on an Ubuntu 22.04 system, so behavior may be unexpected outside of this distro. And if they encounter issues to create an issue with their environment specs.